basic_algs/src/main.rs

```diff
@@ -0,0 +1,66 @@
+fn min(array: &[f64; 10]) -> f64 {
+    let mut cur_min = array[0];
+    for element in array {
+        if cur_min > *element {
+            cur_min = *element
+        }
+    }
+    return cur_min
```

Comment:
Здесь и далее можно не писать `return`:
```suggestion
    cur_min
```
Считается, что в этом случае код "более идиоматичный". Никаких практических недостатков у такого кода, впрочем, нет. В инструменте clippy есть линт для таких случаев: https://rust-lang.github.io/rust-clippy/master/#needless_return